src/Microsoft.AspNet.TestHost/ClientHandler.cs

```diff
@@ -23,14 +24,15 @@ namespace Microsoft.AspNet.TestHost
     /// </summary>
     public class ClientHandler : HttpMessageHandler
```

Comment:
We should be using the default http context factory that sets the accessor for us automatically.